### PR TITLE
Keneucker/feature new updateInfo method

### PR DESCRIPTION
**Pull request by KenEucker**
Adding the updateInfo method and accompanying test file.

This addresses an item from #63.
> Update image information

The upstream merge commits are in here because I created this branch on my forked version of the repository. @kaimallea, let me know if you'd prefer that I create branches on this repo, not my forked repository, for submitting PR's. It might be cleaner that way.

### CODE REVIEW

**Comment by KenEucker** [comment]
I'm also sneaking a typo change in the CONTRIBUTING.md file, "committizen" -> "commitizen". I missed that in my review of that PR, sorry!

File: __tests__/updateInfo.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+const imgur = require('../lib/imgur.js');
```
[the remark this answers is not in the document]
Comment (kaimallea) [reply __tests__/updateInfo.js:1]: I provided a good example in the gallery PR https://github.com/kaimallea/node-imgur/pull/113#issuecomment-795014056

Check the `msw` docs for more deets/examples on how to write handlers to mock out: https://mswjs.io/docs/basics/request-handler

File: lib/imgur.js [hunk 2]
```diff
@@ -350,6 +350,33 @@ imgur.getAlbumInfo = async (id) => {
...
+
+  if (typeof title === 'string' && title.length) {
+    extraFormParams.title = title;
+  }
+
+  if (typeof description === 'string' && description.length) {
+    extraFormParams.description = description;
+  }
+
+  return await imgur._imgurRequest('update', id, extraFormParams);
```
[the remark this answers is not in the document]
Comment (KenEucker) [reply lib/imgur.js:377]: @kaimallea I may have forgotten to unstash, or something. I have written this code several times in my upstream merges but I may have simply been confused. I'm updating this branch now.

File: __tests__/updateInfo.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (KenEucker) [reply __tests__/updateInfo.js:1]: @kaimallea I'm not sure what simplifications you were thinking of. I copied this test from search test, which included these other tests that I thought seemed appropriate enough. I've left all of the tests in there for now.

**Comment by KenEucker** [comment]
@kaimallea this one is ready for another review.

**Review by kaimallea — APPROVED** [review]

**Comment by kaimallea** [comment]
Looks good, thanks!

**Pull request merged**